src/quickstart/provider/minikube/dns.md

```diff
@@ -31,6 +33,34 @@ $ host something-random.192.168.99.100.nip.io
 something-random.192.168.99.100.nip.io has address 192.168.99.100
 ```
 
+### Using DNSMasq
+
+If you `nip.io` is working for you, you can skip this section, and proceed to verify the hostname.
```

Comment:
A very minor grammatical typo here -- perhaps should read `If nip.io is working for you...` (can remove the extra `you`)